examples/src/test/kotlin/BuildersTest.kt

```diff
@@ -10,11 +9,11 @@ import org.bson.BsonObjectId
 import org.bson.Document
 import org.bson.codecs.pojo.annotations.BsonId
 import org.junit.jupiter.api.AfterAll
-import org.junit.jupiter.api.Assertions.*
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.Assertions.assertEquals
```

Comment:
Q: Are both of these needed?

Reply:
no, only the method imports are needed. will clean up